ptf_tests/tests/l3_action_profile_with_vhost_ports.py

```diff
@@ -136,28 +136,33 @@ def runTest(self):
                 print(f"FAIL: Verification of packets sent failed with exception {err}")
                 self.PASSED = False
 
-        # verify whether traffic hits group-2
-        for src in self.config_data['traffic']['in_pkt_header']['ip_src']:
+        sender_vm_id, receiver_vm_id = 0,1
```

Comment:
Same receiver_vm_id for both multicast and broadcast?  There are 4 VMs created based on the log. Which VM will receive unicast, multicast and broadcast?  Please indicate in code and log.

Reply:
we are using genarating only 2 vm's in this tc